test-clients/src/main/java/com/hedera/services/legacy/core/FeeClient.java

```diff
@@ -75,23 +75,17 @@ public static void initialize(int hbarEquiv, int centEquiv, byte[] feeSchBytes)
     }
   }
   public static Map<HederaFunctionality, FeeData> getFeeScheduleMap() {
-    Map<HederaFunctionality, FeeData> feeSchMap = null;
+    Map<HederaFunctionality, FeeData> feeSchMap = new HashMap<>();
     try {
-      if (feeSchMap == null) {
-        feeSchMap = new HashMap<>();
-        Path pathFeeSch =
-            Paths.get(TestHelper.class.getClassLoader().getResource("feeSchedule.txt").toURI());
-        File feeSchFile = new File(pathFeeSch.toString());
-        InputStream fis = new FileInputStream(feeSchFile);
-        byte[] fileBytes = new byte[(int) feeSchFile.length()];
-        fis.read(fileBytes);
-        CurrentAndNextFeeSchedule feeSch = CurrentAndNextFeeSchedule.parseFrom(fileBytes);
-        List<TransactionFeeSchedule> transFeeSchList =
-            feeSch.getCurrentFeeSchedule().getTransactionFeeScheduleList();
-        for (TransactionFeeSchedule transSch : transFeeSchList) {
-          feeSchMap.put(transSch.getHederaFunctionality(), transSch.getFeeData());
-        }
-
+      File feeSchFile = new File("src/main/resource/feeSchedule.txt");
```

Comment:
Just my personal opinion, the original code to get the Path of ".../feeSchedule.txt" should be more flexible. Since this file may soon be refactored out, so the change  as done here is fine. Otherwise, I wouldn't stick to original code to get the Path.